src/components/Grid/Columns/LanguageNameColumn.tsx

```diff
@@ -12,7 +12,10 @@ import { textColumn } from '../ColumnTypes/textColumn';
 export const LanguageNameColumn = <
   const Input extends Merge<
     Partial<GridColDef<Row>>,
-    WithValueGetterReturning<LanguageLookupItem, Row>
+    WithValueGetterReturning<
+      Pick<LanguageLookupItem, 'displayName' | 'name' | 'id'>,
+      Row
+    >
```

Comment:
I don't like picking apart fragments. IMO they should be used 1-1 between GQL-TS.
If columns need a different language shape, then they should declare their own fragment to be used.

Strictly, this is not necessary for this case either, I would just revert.